src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -89,5 +89,46 @@ public async Task<IActionResult> EditEndDate(EditEndDateViewModel viewModel)
             var url = _linkGenerator.ApprenticeDetails(viewModel.AccountHashedId, viewModel.ApprenticeshipHashedId);
             return Redirect(url);
         }
+
+        [Route("{apprenticeshipHashedId}/details/changestatus")]
+        [HttpGet]

```

Comment:
pleaes add the following attribute, I forgot to do it [DasAuthorize(EmployerFeature.ManageApprenticesV2)] to all action methods